profiles/tunables/ros.d/install

```diff
@@ -1,17 +1,13 @@
 # ------------------------------------------------------------------
 #
-#    Copyright (C) 2016 Open Source Robotics Foundation
+#    Copyright (C) 2019 Open Source Robotics Foundation
 #
 #    This program is free software; you can redistribute it and/or
 #    modify it under the terms of version 2 of the GNU General Public
 #    License published by the Free Software Foundation.
 #
 # ------------------------------------------------------------------
 
-# This file is auto-generated. It is recommended you update it using:
-# $ sudo dpkg-reconfigure apparmor
-#
 # The following is a space-separated list of where additional ros install
 # directories are stored. Directories added here are appended to @{ROS_INSTALL}.
-# See ros/home for details.
-@{ROS_INSTALL}=/opt/ros/kinetic
+@{ROS_INSTALL}=/opt/ros/*
```

Comment:
I've updated this profile library wholesale for ROS2, though I'm not sure there is a better why to separate privileges to ros2 versions without hardcoding the distro (which I think is rather fragel and tedious). We could template some things for auto generation during install time, but I'm not sure we need to overcomplicate things. Like what if multiple distros are installed simultaneously, should the debian packages just state they conflict (I'd guess so)? 

However, do folks think we should add the version to the paths and imports, e.g.

``` diff
#include <tunables/global>
- #include <tunables/ros>
+ #include <tunables/ros2>

- profile demo_nodes_cpp/talker @{ROS_INSTALL_LIB}/demo_nodes_cpp/talker {
-  #include <ros/base>
-  #include <ros/node>
+ profile demo_nodes_cpp/talker @{ROS2_INSTALL_LIB}/demo_nodes_cpp/talker {
+  #include <ros2/base>
+  #include <ros2/node>
}
```